DigitalLearningSolutions.Data/DataServices/CentresDataService.cs

```diff
@@ -236,5 +228,29 @@ FROM Centres
             var ipPrefixes = ipPrefixString?.Split(',', StringSplitOptions.RemoveEmptyEntries);
             return ipPrefixes ?? new string[0];
         }
+
+        public IEnumerable<CentreRank> GetCentreRanks(DateTime dateSince, int regionId)
+        {
+            return connection.Query<CentreRank>(
+                @"SELECT 
+                        RANK() OVER (ORDER BY tc.CentreIDCount DESC) AS [Rank],
+                        c.CentreID,
+                        c.CentreName,
+                        tc.CentreIDCount AS [Sum]
+                    FROM 
+	                ( 
+	                    SELECT
+                            Count(c.CentreID) AS CentreIDCount,
+                            c.CentreID
+	                    FROM [Sessions] s 
+	                    INNER JOIN Candidates c ON s.CandidateID = c.CandidateID 
+	                    INNER JOIN Centres ct ON c.CentreID = ct.CentreID
+	                    WHERE s.LoginTime > @dateSince AND c.CentreID <> 101 AND (ct.RegionID = @RegionID OR @RegionID = -1)

```

Comment:
I don't think the CentreID was part of the ACs.. I will check with Kevin whether this is needed in the new version.